internal/sdkmath/floor.go

```diff
@@ -1,5 +1,3 @@
-// +build go1.10
-
 package sdkmath
 
 import "math"
```

Comment:
I think we can  remove this file as well, and use the stdlib version directly in the SDK's code.